cep-0006.md

```diff
@@ -200,12 +196,11 @@ We do not expect any backwards compatibility issues for this new feature.
   it makes more sense to put these in their own file. Plus, it allows more flexibility for dynamic routing options
   to this file if that becomes necessary in the future.
 
-
 ## References
 
 This implementation is similar to the way that `npm` handles version update reminders:
 
-```
+```bash
```

Comment:
Do we always need a language type for code blocks? Sometimes GitHub lacks an appropriate language type for particular code

Reply:
Yes. Marking text is fine.